### PR TITLE
Feature/adding functionality to qa dashboard

### PR DESCRIPTION
**Pull request by sajidali36**

### CODE REVIEW

File: app/views/bugs/index.html.erb [hunk 1]
```diff
@@ -20,14 +20,15 @@
         <td><%= bug.description %></td>
         <td><%= bug.deadline %></td>
         <td><%= bug.project_id %></td>
-        <td><%= link_to 'Show', bug %></td>
-        <td><%= link_to 'Edit', edit_bug_path(bug) %></td>
-        <td><%= link_to 'Destroy', bug, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to 'Show Project', [@project] %></td>
```
Comment (MIbtehajNasar) [inline app/views/bugs/index.html.erb]: kindly use `project_path`

File: app/views/bugs/index.html.erb [hunk 2]
```diff
@@ -20,14 +20,15 @@
...
+        <td><%= link_to 'Edit Bug', edit_project_bug_path(@project, bug) %></td>
+        <td><%= link_to 'Destroy Bug', [@project, bug], method: :delete, data: { confirm: 'Are you sure?' } %></td>
```
Comment (MIbtehajNasar) [inline app/views/bugs/index.html.erb]: same as above use path instead of passing object

File: app/views/projects/show.html.erb [hunk 3]
```diff
@@ -10,5 +10,14 @@
   <%= @project.description %>
 </p>
 
+<h2>Bugs</h2>
+<% for bug in @project.bugs %>
```
Comment (MIbtehajNasar) [inline app/views/projects/show.html.erb]: kindly use each loop here

**Review by MIbtehajNasar — CHANGES_REQUESTED** [review]

File: app/views/projects/show.html.erb (hunk 3 above)
Comment (sajidali36) [reply app/views/projects/show.html.erb]: done

File: config/routes.rb [hunk 4]
```diff
@@ -5,13 +5,15 @@
...
+  resources :projects do
+    resources :bugs
+  end
   resources :projects do
     member do
       get 'edit_users_list'
     end
   end
-
+  post "projects/:id/bugs/new", to: "bugs#create"
```
Comment (MIbtehajNasar) [inline config/routes.rb]: why do we need this here?

**Pull request merged**